### DISCUSSION TITLE
Public language dictionary for teams

### DISCUSSION THREAD

**Discussion by KevinKrueger** [EWSoftware/VSSpellChecker#229] [Ideas]
Hey Dear SpellChecker Team ;)

In my idea it is about creating in the previously added words from my comments a for example internal location on a SMB or similar, where then when changes, the local dictionary is synchronized, so that everyone can use the current last added words in his CodeEditor.

If you have any questions, just contact me.
Would be very useful, especially for teams.

Kind regards,
Kevin Krüger

**Comment by EWSoftware** [comment]
You can add a configuration file to the solution or project and specify in it to use a user dictionary stored in the solution/project.  That allows for source controlled user dictionaries per solution/project.  In the global configuration or in a solution/project configuration file, you can specify a shared location for the user dictionaries.  That would allow you to put them on a network share accessible to all team members and allows the user dictionaries to be backed up.  The downside there is that they cannot be source controlled and there is a small risk that you may lose words if two people happen to save a word to it at the same time.  As long as you are using the latest release, that risk is minimized as changes were made to help prevent that.

See the User Dictionary section of the [Dictionary Settings](http://ewsoftware.github.io/VSSpellChecker/html/af34b863-6a1c-41ed-bcf2-48a714686519.htm#UserDictionary) help topic for more information.
